app/javascript/toolbar-actions/delete.js

```diff
@@ -1,18 +1,16 @@
 import { API } from '../http_api';
 
-export function showMessage(messages, success) {
-  if (typeof messages === 'string') {
-    add_flash(messages, success ? 'success' : 'error');
-  } else {
-    Object.keys(messages).forEach((msgStatus) => {
-      const statusKey = msgStatus === 'true';
-      if (statusKey && messages[statusKey] > 0) {
-        add_flash(sprintf(__('Deleting of %s items queued.'), messages[statusKey]), 'success');
-      } else if (messages[statusKey] > 0) {
-        add_flash(sprintf(__('Failed to delete %s items.'), messages[statusKey]), 'error');
-      }
-    });
-  }
+export function showMessage(messages, labels = {single: '', multiple: ''}) {
+  Object.keys(messages).forEach((msgStatus) => {
+    const statusKey = msgStatus === 'true';
+    if (statusKey && messages[statusKey] > 0) {
+      const label = messages[statusKey] === 1 ? labels.single : labels.multiple;
+      add_flash(sprintf(__('Delete initiated for %s %s.'), messages[statusKey], label), 'success');
```

Comment:
same here, possibly `n__` but not sure how to do that when the key is coming from elsewhere.